Util.java

```diff
@@ -1,5 +1,6 @@
 package org.usfirst.frc4904.standard;
 
+
```

Comment:
yes that's valid when i had my nested ternary pull request it adde the same newline